src/main/webapp/app/assessment/assessment-header/assessment-header.component.html

```diff
@@ -8,14 +8,23 @@ <h3 class="top-container">
     <div class="row-container" *ngIf="!isLoading">
         <span
             id="assessmentLocked"
-            *ngIf="!isAssessor"
+            *ngIf="!isAssessor && !isComplaint"
             class="text-danger ml-3"
             style="font-size: 65%;"
             jhiTranslate="artemisApp.assessment.assessmentLocked"
             [translateValues]="{ otherUser: result?.assessor?.name + ' (' + result?.assessor?.login + ')' }"
         >
             Assessment locked by another user!
         </span>
+        <span
+            *ngIf="!isAssessor && isComplaint"
+            class="ml-3"
+            style="font-size: 65%;"
+            jhiTranslate="artemisApp.assessment.assessmentLockedComplaintView"
+            [translateValues]="{ otherUser: result?.assessor?.name + ' (' + result?.assessor?.login + ')' }"
+        >
+            Assessment locked by another user!
+        </span>
```

Comment:
I think you can merge this lines and prevent a lot of duplicated code as you are using the `*ngIf` only to differentiate the translation strings. You can place the conditional operator inline with the translation string (`{{ (isComplaint ? 'artemisApp.assessment.assessmentLockedComplaintView' : 'artemisApp.assessment.assessmentLocked') | translate}}`) and keep `*ngIf="!isAssessor"` on the `span`element.

If I see this correctly, the id (`id="assessmentLocked"`) is not used anywhere either. So you can remove it.

Reply:
Hmm.. and what about the class ? it is different, so I will have to use a conditional operator there as well. I thought is might be more readable to have another span element than putting conditional statements for each value.

Reply:
Sorry, haven't seen that... was a little late.